src/app/components/vertical-icons-component.js

```diff
@@ -57,7 +58,10 @@ class VerticalIconComponent {
    * @param {ModuleProfile} profile The profile of the module
    */
   addIcon ({kind, name, icon, displayName}) {
-    this.icons[name] = yo`<div class="${css.icon}" onclick="${(e) => { this._iconClick(name) }}" plugin="${name}" title="${displayName || name}" ><img src="${icon}" alt="${name}" /></div>`
+    this.icons[name] = yo`
+      <div class="${css.icon}" onclick="${(e) => { this._iconClick(name) }}" plugin="${name}" title="${displayName}" >
```

Comment:
on the diff, the previous version is `title="${displayName || name}"`

Reply:
I know. I thought we should not show name. The description should be provided, if not we don't show anything. no?

Reply:
I think we have to default to something

Reply:
ok